youtube_dl/extractor/tasvideos.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class TasVideosIE(InfoExtractor):
```

Comment:
[The website of TASVideos](http://tasvideos.org) writes the name with uppercase `TAS`
```suggestion
class TASVideosIE(InfoExtractor):
```

Reply:
Thanks!  
You are right, I got too caught up in the camelCase convention there.
I changed the class name, import statement and the pull-request's name accordingly~